packages/webapp/src/components/Animals/AddAnimalsFormCard/styles.module.scss

```diff
@@ -0,0 +1,35 @@
+.form {
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+}
+
+.formHeader {
+  display: flex;
+  justify-content: space-between;
+}
+
+.removeBtn {
+  display: flex;
+  align-items: center;
+  gap: 4px;
+  border-radius: 16px;
+  padding: 2px 9px;
```

Comment:
Let's go with `4px 8px`. If I'm not mistaken, we would like to use multiples of 4px, even if it differs slightly from the styles in the mockups.